src/components/signpost.js

```diff
@@ -30,51 +68,20 @@ export const directions = [
     url: "https://discord.gg/awTtjcQ9Us",
     description: "Fast-paced MetaGame.",
   },
-  {
-    emoji: "📝",
-    label: "Forum",
-    url: "https://forum.metagame.wtf/",
-    description: "Thoughtful MetaGame.",
-  },
   {
     emoji: "🐤",
     label: "Twitter",
     url: "https://twitter.com/metafam",
     description: "Follow MetaGame on Twitter.",
   },
   {
-    emoji: "🗺️",
-    label: "Raids",
-    url: "/docs/enter-metagame/navigation-board",
-    description: "Find raids that peak your interest.",
-  },
-  {
-    emoji: "🛠️",
+    emoji: "😺",
```

Comment:
Why the change from ⚔️  to a 🐱?

Reply:
github logo is a octocat. i was just playing around and i thought the current icons needed improvement.